server/initial_data.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+import subprocess
+from server.mbta_api import routes_info, vehicle_data_for_routes, stops_for_route
+from server.last_seen import get_recent_sightings_for_lines
+
+def get_git_tag():
```

Comment:
Nothing has changed about the initial data loading. I had to move it to its own module because of a circular dependency between `mbta_api` and `last_seen`.